src/CaptureFileInfo/Manager.cpp

```diff
@@ -48,14 +48,14 @@ void Manager::SaveCaptureFileInfos() {
   settings.endArray();
 }
 
-void Manager::AddOrTouchCaptureFile(const QString& path) {
+void Manager::AddOrTouchCaptureFile(const std::filesystem::path& path) {
```

Comment:
I don't see why you're doing this. Maybe you could add an explanation to the commit message?

Reply:
I will add an explanation to the commit message when doing the rebase.